src/app/core/admin/admin-entity-details/admin-entity-field/configure-entity-field-validator/configure-entity-field-validator.component.scss

```diff
@@ -0,0 +1,9 @@
+.validator-form-cell {
+    display: flex;
+    flex-direction: column;
+    // set the width of each form field to 100% in every form component that is a descendent
+    // of the columns-wrapper class
+    mat-form-field {
+      width: 80%;
+    }
+}
```

Comment:
for a flex layout, you can use our global class "flex-column" without having to define your own css